src/Main.java

```diff
@@ -13,5 +13,96 @@ public static void main(String[] args) {
             default:
                 System.out.println("Unknown");
         }
+        System.out.println("Задача N2");
+//Задача 2 Чтобы не плодить переменные переменная clientOS взята из первой задачи
+        int clientOs = 1;
```

Comment:
```suggestion
        clientOS = 1;
```
если не хотел плодить переменные, то нужно было сделать вот так